packages/api-client/tests/secret.spec.ts

```diff
@@ -10,9 +11,8 @@ describe('Secret Controller Tests', () => {
   const email = 'johndoe@example.com'
   let projectSlug: string | null
   let workspaceSlug: string | null
-  let environmentSlug: string | null
   let secretSlug: string | null
-  let environment
+  let environment: Environment
```

Comment:
Remove this line, and keep `environmentSlug`. Make all the comparisons on `environmentSlug`. We don't want to compare ID anymore